index.html

```diff
@@ -184,5 +264,102 @@ <h4 class="top-banner__subtitle">
         </div>
       </div>
     </section>
+    <section class="why-us">
+      <div class="why-us__container">
+        <div class="why-us__main-content">
+          <h2 class="why-us__title">WHY CHOOSE US</h2>
```

Comment:
As you can see you have same title and subtitle designs in all sections on the design. So it makes sense to give them generic class names that you could use in future sections. 
`section__title` and `section__subtitle` sound pretty good imho.

Reply:
Main idea is to find typical elements that are the same all over the page and give them generic but meaningful class  names. And style them just once.